packages/reason-relay/src/ReactExperimental.re

```diff
@@ -4,13 +4,6 @@ type suspenseConfig = {
   busyMinDurationMs: option(int),
 };
 
-[@bs.module "react"]
-external _useTransition: suspenseConfig => ((unit => unit) => unit, bool) =
-  "useTransition";
-
-let useTransition = (~timeoutMs, ~busyDelayMs=?, ~busyMinDurationMs=?, ()) =>
-  _useTransition({timeoutMs, busyDelayMs, busyMinDurationMs});
-
 [@bs.module "react"]
 external _useDeferredValue: ('value, suspenseConfig) => 'value =
```

Comment:
Out of curiosity - the new bindings from `reason-react` don't include bindings for `useDeferredValue` I take it?

Reply:
Yes, `useDeferredValue` is not in the bindings yet.